instrumentation/log4j/log4j-mdc-1.2/javaagent/src/test/groovy/Log4j1MdcTest.groovy

```diff
@@ -33,6 +33,8 @@ class Log4j1MdcTest extends AgentInstrumentationSpecification {
     events[1].getMDC("trace_id") == null
     events[1].getMDC("span_id") == null
     events[1].getMDC("trace_flags") == null
+
+    assertTraces(0) {}
```

Comment:
what's this assertion?

Reply:
I guess it's not necessary, just made sure there are zero traces in this test